mu4/domain/notation/imasternotation.h

```diff
@@ -0,0 +1,47 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FIT-0NESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_DOMAIN_IMASTERNOTATION_H
+#define MU_DOMAIN_IMASTERNOTATION_H
+
+#include "modularity/imoduleexport.h"
+#include "inotation.h"
+#include "ret.h"
+#include "io/path.h"
+
+namespace mu {
+namespace domain {
+namespace notation {
+class IMasterNotation : virtual public INotation
+{
+public:
+    virtual Ret load(const io::path& path) = 0;
+    virtual io::path path() const = 0;
+
+    virtual Ret createNew(const ScoreCreateOptions& scoreInfo) = 0;
+
+    virtual std::vector<INotationPtr> parts() const = 0;
```

Comment:
Maybe better add IExcerptNotation interface?, even if it be empty

Reply:
I do not understand its purpose. I can change my mind if you tell me the advantages of this empty interface

Reply:
I see benefits like this:
* Semantics - we will see where Excerpt is returned or expected, and not any notation.
* If there are methods that relate only to Excerpt, then they can be added there in future. Otherwise, I am afraid for the sake of such a method they will not create a class and change a lot of code, and they will add this special method to the base class.

Reply:
fixed